### PR TITLE
Remove unneeded Cuda Half include

### PR DESCRIPTION
**Pull request by brian-kelley**
Kokkos recently reorganized half precision related headers, breaking
this include. It wasn't necessary anyway.

### CODE REVIEW

**Comment by ndellingwood** [comment]
I think this duplicates #1281; seems PR testing is blocked until #1278 is merged

**Review by ndellingwood — APPROVED** [review]

**Comment by brian-kelley** [comment]
Oops, I didn't see that. I'll just close this.

**Pull request closed without merging**
